dndwiki/classes/class-necromancer.xml

```diff
@@ -1,565 +1,611 @@
+			<select type="Spell" name="Spell (Necromancer)" supports="$(spellcasting:list), $(spellcasting:slots)" level="13" spellcasting="Necromancer" />
+			<select type="Spell" name="Spell (Necromancer)" supports="$(spellcasting:list), $(spellcasting:slots)" level="15" spellcasting="Necromancer" />
+			<select type="Spell" name="Spell (Necromancer)" supports="$(spellcasting:list), $(spellcasting:slots)" level="17" spellcasting="Necromancer" />
+		</rules>
+	</element>
+
+	<element name="Life Tap" type="Class Feature" source="D&amp;D Wiki" id="ID_WIKI_CLASS_FEATURE_NECROMANCER_LIFE_TAP">
+		<description>
+			<p>Starting at 1st level, your touch can siphon the life force of others to bolster your own. As an action, you can make a melee spell attack against a living creature, dealing necrotic damage equal to 1d8 + your Charisma modifier on a hit. You gain temporary hit points equal to the amount of necrotic damage dealt. If this feature kills the creature, you gain twice as many temporary hit points from using this feature. The damage increases by 1d8 when you reach 5th level (2d8), 11th level (3d8), and 17th level (4d8). You may use this feature a number of times equal to your Charisma modifier(minimum one) and you regain all uses of this feature at the end of a long rest.</p>
+		</description>
+		<sheet>
 			<description>{{life tap:count}}/long rest. As an action, you can make a melee spell attack against a living creature, dealing necrotic damage equal to 1d8 + your Charisma modifier on a hit. You gain temporary hit points equal to the amount of necrotic damage dealt. If this feature kills the creature, you gain twice as many temporary hit points from using this feature.</description>
 			<description level="5">{{life tap:count}}/long rest. As an action, you can make a melee spell attack against a living creature, dealing necrotic damage equal to 2d8 + your Charisma modifier on a hit. You gain temporary hit points equal to the amount of necrotic damage dealt. If this feature kills the creature, you gain twice as many temporary hit points from using this feature. </description>
 			<description level="11">{{life tap:count}}/long rest. As an action, you can make a melee spell attack against a living creature, dealing necrotic damage equal to 3d8 + your Charisma modifier on a hit. You gain temporary hit points equal to the amount of necrotic damage dealt. If this feature kills the creature, you gain twice as many temporary hit points from using this feature.</description>
 			<description level="17">{{life tap:count}}/long rest. As an action, you can make a melee spell attack against a living creature, dealing necrotic damage equal to 4d8 + your Charisma modifier on a hit. You gain temporary hit points equal to the amount of necrotic damage dealt. If this feature kills the creature, you gain twice as many temporary hit points from using this feature.</description>
```

Comment:
<sheet> can be changed to <sheet action="Action" usage="Long Rest" /> as this class was made before this can be allowed.